lmfit/model.py

```diff
@@ -993,3 +993,26 @@ def plot(self, datafmt='o', fitfmt='-', initfmt='--', numpoints=None,
                             fit_kws=fit_kws, ax_kws=ax_res_kws)
 
         return fig
+
+    def aic(self):
```

Comment:
This would be better as a property:

    @property
    def aic(self):
        # return AIC

In that way you get the AIC value as `model.aic` rather than `model.aic()`